src/lastRevision.sh

```diff
@@ -16,6 +16,6 @@
 
 # Find the current revision, store it in a file
 FILE=$1
-LASTREV=`svn info | grep '^Revision' | sed -e 's/Revision: *//'`
+LASTREV=`git rev-parse HEAD | cut -c1-8`
```

Comment:
Thanks @eribeiro for the patch. Any specific reason to use only the first 8 characters of the git SHA-1. Will this be helpful in analyzing whether a particular fix included in the release?

How about keeping the complete SHA-1?